src/Application/Matching/Adapter.php

```diff
@@ -235,29 +222,12 @@ private function buildKey(CampaignFunding $funding): string
         return "fund-{$id}-available-opt";
     }
 
-    /**
-     * After completing fund allocation, update the database funds available to our last known values, without locks.
-     * This is not guaranteed to *always* be a match for the real-time Redis store since we make no effort to fix race
-     * conditions on the database when using Redis as the source of truth for matching allocation.
-     */
-    public function saveFundingsToDatabase(): void
-    {
-        $this->entityManager->wrapInTransaction(function () {
-            foreach ($this->fundingsToPersist as $funding) {
-                $this->entityManager->persist($funding);
-            }
-        });
-    }
-
     /**
      * @psalm-param numeric-string $newValue
      */
     private function setFundingValue(CampaignFunding $funding, string $newValue): void
```

Comment:
`setFundingValue` looks like it might as well be inlined to its callers now. The name isn't any more meaningful that `setAmountAvailable` is it?

Reply:
Nope, good spot. Done in ae8a1b145b7d69e787e9fb66be5522a02afb32d8